docs/pages/api/renderers/web.md

```diff
@@ -2,6 +2,7 @@ import Docs from "@site/pages/api/generated/renderer-WebRenderer.md"
 
 # Web Renderer
 
-Represents an instance of a website opened with Chromimum embeded inside the compositor. Used by a [`WebView` component](../components/WebView). Only one `WebView` component can use a specific instance at a time.
+Represents an instance of a website opened with Chromium embeded inside the compositor. Used by a [`WebView` component](../components/WebView). Only one `WebView` component can use a specific instance at a time.
+Before the web renderer can be used, you need to make sure that compositor version which includes `"web renderer feature"` is used, and the environment variable `LIVE_COMPOSITOR_WEB_RENDERER_ENABLE` is enabled.
```

Comment:
Do not use word feature, this is rust/cargo specific term. You can refer to is as "compositor with web renderer" / "compositor with web rendering support" or sth like that

Whathever you decide make sure to use it consistentlly.